index.html

```diff
@@ -46,92 +46,8 @@ <h1>Abdel-Khafid SALAOU</h1>
    </section>
```

Comment:
- [ ] Please as the last reviewer said, there are a few adjustments you need to make to align the buttons and text accordingly, kindly shift the button and "previous project" to be aligned with the paragraph above just as in the template. This gives a better user experience from a user point of view and helps you stick to requirements.
![image](https://user-images.githubusercontent.com/74744787/224502622-ed7bacba-9ca7-46da-86e9-e542ff0d432f.png)

Reply:
![Captufinal](https://user-images.githubusercontent.com/119133987/224502912-316f1f89-53bb-407f-8bbd-cc56f7a462cc.PNG)
I'm confused. its perfectly aligned on my side